components/brave_rewards/resources/page/promos/index.tsx

```diff
@@ -58,11 +58,15 @@ const getLink = (type: PromoType) => {
   return ''
 }
 
+const getRootImagePath = (path: string) => {
+  return `/${path}`
```

Comment:
We'll need a more generic solution for this problem (since it affects other resource loads when on a rewards page subpath), but this is fine for now. Can we add a comment explaining why this is necessary (and perhaps how we need a better solution in the future)?

Reply:
Yep, good idea - done.

Reply:
Perfect, thanks!